### ISSUE TITLE
Not connecting

### ISSUE THREAD

**Issue by AlanFPS** [liximomo/vscode-sftp#840]
Hey guys, how's it going? I'm quite new so be patient please haha.

I've tried a couple of days ago to use this extension (following [this guide]( https://artisansweb.net/ftp-sftp-extension-for-visual-studio-code/)) and I got some errors so I've found #613 where some people explained that apparently there's an issue with the absolute path for FTP or SFTP (also someone mentioned #356).

In any case, at that time I forgot about it and now I'm trying to connect again but it just keeps "thinking" and trying to connect endlessly. No error messages, no nothing.

Any ideas? Thanks! :)

**Comment by daveteu** [comment]
same here. did u just update VS studio code too?

**Comment by AlanFPS** [comment]
> same here. did u just update VS studio code too?

yep, before the update I couldn't connect but at least I received the error messages.

**Comment by lacoste-sochi** [comment]
The problem is the same...

the project doesn't load, only its folders ...
